arm/Microsoft.ContainerRegistry/registries/deploy.bicep

```diff
@@ -158,6 +148,15 @@ param diagnosticEventHubName string = ''
 @description('Optional. The name of the diagnostic setting, if deployed.')
 param diagnosticSettingsName string = '${name}-diagnosticSettings'
 
+@description('Optional. The resource ID of a key vault to reference a customer managed key for encryption from.')
```

Comment:
```suggestion
@description('Optional. The resource ID of a key vault to reference a customer managed key for encryption from. Requires the \'acrSku\' to be \'Premium\'.')
```

Reply:
Updated with slight adjustment (hence not used the commit).